src/auth/auth.service.ts

```diff
@@ -337,11 +347,20 @@ export class AuthService {
       return userDto;
     } catch (error) {
       this.logger.error('Login Error: ', error);
-      throw new EaseyException( new Error(`Login Error: ${error.message}`), HttpStatus.BAD_REQUEST,
+      throw new EaseyException(new Error(`Login Error: ${error.message}`), HttpStatus.BAD_REQUEST,
       );
     }
   }
 
+  async validateUserMaintenanceAblity(userDto: UserDTO) {
```

Comment:
This doesn't need to be asynchronous:
```suggestion
  validateUserMaintenanceAblity(userDto: UserDTO) {
```